core/tooltip.js

```diff
@@ -32,102 +33,104 @@ goog.require('Blockly.utils.string');
  * eventually unwinds to a string.
  * @typedef {string|{tooltip}|function(): (string|!Function)}
  */
-Blockly.Tooltip.TipInfo;
+let TipInfo;
+exports.TipInfo = TipInfo;
 
 /**
  * Is a tooltip currently showing?
  */
-Blockly.Tooltip.visible = false;
+let visible = false;
+exports.visible = visible;
```

Comment:
This doesn't look correct.  If we're exporting `visible`, presumably external code wants to know the _current_ value of the flag, not just the initial value.

See [the style guide](https://google.github.io/styleguide/jsguide.html#es-module-exports) under "Mutability of exports" for how to deal with this (but maybe for now `exports.visible` needs to be an actual getter, for backwards compatibility.)

(Also: perhaps add `@type {boolean}` to the JSDoc?)

Reply:
Updated this (and DIV) following the pattern we've settled on.